app/doctor/ntp.go

```diff
@@ -0,0 +1,105 @@
+package doctor
+
+import (
+	"context"
+	"runtime"
+	"time"
+
+	"github.com/beevik/ntp"
+	"github.com/fatih/color"
+	"github.com/spf13/viper"
+
+	"github.com/iyear/tdl/pkg/consts"
+)
+
+func getDefaultNTPServers() []string {
```

Comment:
ntp a cross-platform protocol, only a single list is needed.

Reply:
The reason I provide different NTP lists for windows, linux and macOS here is that the default NTP servers for these three operating systems are different. macOS uses times.apple.com, windows use times.windows.com, while linux varies by distribution -- which is why I selected ntp.ubuntu.com as the default first try.

As for why I use a list of NTP servers instead of a single one, it’s because during my testing, I found that connections to time.apple.com were quite unstable.

If you believe that a unified NTP server list should be used across all platforms, what order do you think the NTP servers should follow?

Reply:
Either is fine. If Apple's ntp servers are unstable, then don't include it.

Reply:
@iyear I understand your point about unifying the NTP list. However, `time.apple.com` is the default NTP server on macOS and rarely changed by users. Using platform defaults (time.apple.com for macOS, time.windows.com for Windows) ensures we're using servers that are already working for users' systems, providing better latency and reliability.

The `time.apple.com` unstable issue I mentioned can be handled with fallbacks. I'd prefer keeping the platform-specific approach as it respects each OS's defaults, but I'm happy to change it if you feel strongly about it.